bin/sparqlalgebrajs.ts

```diff
@@ -26,5 +26,5 @@ if (args.r || args.reverse)
 }
 else
 {
-    console.log(JSON.stringify(Util.objectify(translate(args._[0], { quads: args.q })), null, 2));
-}
\ No newline at end of file
+    console.log(JSON.stringify(Util.objectify(translate(args._[0], { quads: args.q, sparqlStar: true })), null, 2));
```

Comment:
Is there a use case for users who might not want this option? In that case we should make it an option that can be chosen.

Reply:
I followed the same reasoning as SPARQL.js to enable it by default on the CLI.
We maybe could add a `--strict` flag to disable it?
https://github.com/RubenVerborgh/SPARQL.js/#standalone

Reply:
That sounds good. It should probably also be added in the if-call above right?

Reply:
Ah yes, missed that one.